en/SUMMARY.md

```diff
@@ -538,8 +537,7 @@
       * [wind](msg_docs/wind.md)
       * [yaw_estimator_status](msg_docs/yaw_estimator_status.md)
     * [MAVLink Messaging](middleware/mavlink.md)
-    * [RTPS/DDS Interface: PX4-Fast RTPS(DDS) Bridge](middleware/micrortps.md)
-      * [Manually Generate Client/Agent](middleware/micrortps_manual_code_generation.md)
```

Comment:
We may need to replace this link with a new doc. Could well have the same name. But will reflect microDDS.